assets/js/admin.js

```diff
@@ -0,0 +1,99 @@
+function isAccessTokenSet() {
+    return goostavApplicationConfig.accessToken && goostavApplicationConfig.accessToken !== '';
```

Comment:
I think the second check is redundant, an empty string is falsy https://developer.mozilla.org/en-US/docs/Glossary/Falsy